spec/classes/init_spec.rb

```diff
@@ -364,6 +364,17 @@
         it_behaves_like 'bool value'
         it_behaves_like 'non enabled feature'
       end
+
+      context 'always_auto_reboot_at_scheduled_time_minutes =>' do
+        let(:reg_key) { "#{au_key}\\AlwaysAutoRebootAtScheduledTimeMinutes" }
+        let(:param_sym) { :always_auto_reboot_at_scheduled_time_minutes }
+        let(:below_range) { 14 }
+        let(:above_range) { 181 }
```

Comment:
14 is the first error below range, 181 is the first value above range.

It looks like you set the tests up correctly. I'm not sure what would have caused the failure straight away.